### ISSUE TITLE
Installation on Linux fails when /proc is a read-only filesystem.

### ISSUE THREAD

**Issue by ndouglas** [lando/lando#2880]
**Tell us about your setup**

Lando 3.0.26 on Ubuntu 20.10 running in a LXC container on Proxmox.

**Tell us about the command you were running**
`sudo dpkg -i lando-v3.0.26.deb` :grin:

**Tell us about the error you got**
```
(Reading database ... 28359 files and directories currently installed.)
Preparing to unpack lando-v3.0.26.deb ...
Unpacking lando (3.0.26) over (3.0.26) ...
Setting up lando (3.0.26) ...
+ LANDO_ROOT=/usr/share/lando
+ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/usr/local/bin
+ TERM=xterm
+ chmod 755 -R /usr/share/lando
+ mkdir -p /usr/local/bin
+ ln -sf /usr/share/lando/bin/lando /usr/local/bin/lando
++ awk -F: '$1 == "sudo" {print $4}' /etc/group
+ SUDOERS=nathan
+ '[' -f /proc/sys/vm/max_map_count ']'
+ sysctl -w vm.max_map_count=262144
sysctl: setting key "vm.max_map_count": Read-only file system
dpkg: error processing package lando (--install):
 installed lando package post-installation script subprocess returned error exit status 255
Processing triggers for mime-support (3.64ubuntu1) ...
Errors were encountered while processing:
 lando
```

**Tell us generally about your bug**
`installer/linux/scripts/postinst` assumes that if the `/proc/sys/vm/max_map_count` file exists, that it can be written.  This isn't true in this (possibly pathological, certainly edge) case 🙂   In the case of running on something like Proxmox, `/proc` will be mounted into the LXC container read-only.  

The modification to `vm.max_map_count` could be done directly to the host.  Unfortunately, even when this is done, the installation still fails because the attempt to change the kernel parameter is made regardless of whether the change is necessary.

I'll open a PR shortly that'll compare the current value of `vm.max_map_count` to the desired value of 262144 and attempt to modify the parameter only if it is lower than that value.  That should allow me and other people running setups of questionable wisdom to use Lando 😁

**Comment by blue928** [comment]
I can report an identical issue. I've been developing more on Debian 10 via ChromeOS and dealing with related file system configuration quirks. 

Preparing to unpack lando-stable.deb ...
Unpacking lando (3.0.26) ...
Setting up lando (3.0.26) ...
+ LANDO_ROOT=/usr/share/lando
+ PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/bin
+ TERM=xterm
+ chmod 755 -R /usr/share/lando
+ mkdir -p /usr/local/bin
+ ln -sf /usr/share/lando/bin/lando /usr/local/bin/lando
++ awk -F: '$1 == "sudo" {print $4}' /etc/group
+ SUDOERS=bluepresley
+ '[' -f /proc/sys/vm/max_map_count ']'
+ sysctl -w vm.max_map_count=262144
sysctl: permission denied on key 'vm.max_map_count'
dpkg: error processing package lando (--install):
 installed lando package post-installation script subprocess returned error exit status 255
Processing triggers for desktop-file-utils (0.23-4) ...
Processing triggers for mime-support (3.62) ...
Errors were encountered while processing:
 lando

**Comment by ndouglas** [comment]
This was closed by #2881, my bad.
